src/main/java/myfirstproject/dao/DataBaseDao.java

```diff
@@ -0,0 +1,12 @@
+package myfirstproject.dao;
+
+import java.util.Map;
+import myfirstproject.model.Fruit;
+
+public interface DataBaseDao {
```

Comment:
Make your names as specific as possible 
```suggestion
public interface FruitDao {
```